app/views/products/index.erb

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+  <div class="row">
+    <table class="table table-hover">
+      <thead class="thead-default">
+        <tr>
+          <th width="5%" class="text-xs-right">ID</th>
+          <th>Name</th>
+          <th width="10%" class="text-xs-right">Price</th>
+          <th width="10%" class="text-xs-right">VAT</th>
```

Comment:
I don't think there's one for `<table>`. I'm not dealing with their based-on-floats-grid layout but with somewhat fixed `<table>` cells, so classes like `.col-sm-x` and the like aren't applicable here. At least that's what I think you're referring to.

I could make the table rows stackable and responsive, but I don't really want to invest time in UI since it's probably gonna change a lot soon.